.github/workflows/e2e.maven.workflow_dispatch.main.default.slsa3.yml

```diff
@@ -0,0 +1,101 @@
+on:
+  - workflow_dispatch
+# TODO: Un-comment this     
+#  schedule:
+#    - cron: "0 6 * * *"
+#  workflow_dispatch:
+#
+permissions: read-all
+
+concurrency: "e2e-maven-workflow_dispatch-main-default-slsa3"
+
+env:
+  GH_TOKEN: ${{ secrets.E2E_GO_TOKEN }}
+
+jobs:
+  bootstrap:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
```

Comment:
`boostrap` shouldn't run when `input_trigger` is `true`. See the Node.js builder test for an example.
https://github.com/slsa-framework/example-package/blob/bee082ddeb9ab63ffebd5808383308424fb01175/.github/workflows/e2e.nodejs.workflow_dispatch.main.default.slsa3.yml#L26C13-L26C111

Reply:
Fixed in https://github.com/slsa-framework/example-package/pull/253/commits/dcdf6caa5d7154b91528c85e577c48731690cbe0.